kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ProxyConfigConfigMap.java

```diff
@@ -191,8 +226,7 @@ private static String filterDefinitionName(FilterRef filterCrRef) {
     }
 
     @NonNull
-    private List<NamedFilterDefinition> filterDefinitions(Context<KafkaProxy> context, VirtualKafkaCluster cluster, ResolutionResult resolutionResult)
-            throws InvalidClusterException {
```

Comment:
was the `throws InvalidClusterException` just some dead code?

Reply:
It was dead once I removed the check for conflicts.